iroh-net/src/dns/node_info.rs

```diff
@@ -69,15 +99,15 @@ pub fn from_z32(s: &str) -> Result<NodeId> {
     Ok(node_id)
 }
 
-/// Node info contained in a DNS _iroh TXT record.
+/// Information about and iroh node which is contained in an `_iroh` TXT resource record.
```

Comment:
```suggestion
/// Information about the iroh node which is contained in an `_iroh` TXT resource record.
```

or maybe you meant `an iroh node`?

Also, same comment about `IROH_TXT_NAME`

Reply:
yeah, i means `an iroh node`.  but `the iroh node` also works